RxSwiftExample/Models/Profile/ProfileModel.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  ProfileModel.swift
+//  RxSwiftExample
+//
+//  Created by Johnny Yen on 2020/5/8.
+//  Copyright © 2020 Test. All rights reserved.
+//
+
+import Foundation
+import CoreData
+
+struct Profile: Equatable {
+
+    private var uuid: String = UUID().uuidString
+    var name: String
+    var birthday: String
+
+    init() {
+        self.name = ""
+        self.birthday = ""
+    }
+
+    init(name: String, birthday: String, uuid: String = UUID().uuidString) {
+        self.name = name
```

Comment:
If a **uuid** will be assigned no matter what, maybe you can change the passed-in _uuid_ parameter as an optional String.